packages/components/src/combobox-control/types.ts

```diff
@@ -74,4 +74,11 @@ export type ComboboxControlProps = Pick<
 	 * The current value of the control.
 	 */
 	value?: string | null;
+
+	/**
+	 * Automatically expand the dropdown when the control is focused.
```

Comment:
```suggestion
	 * Automatically expand the dropdown when the control is focused.
	 * If the control is clicked, the dropdown will expand regardless of this prop.
```

I think this will be a cause of confusion if we don't mention this explicitly. Also, it will help clarify the intended expansion behavior if we decide to rewrite the underlying implementation of this v1 component with Ariakit someday.